tests/visual/visuals.test.ts

```diff
@@ -0,0 +1,59 @@
+import glob from 'glob';
+import path from 'path';
+import { renderTest } from './tester';
+
+const paths = glob.sync('**/*.scene.ts', { cwd: path.join(process.cwd(), './tests') });
+const scenes = paths.map((p) =>
+{
+    const relativePath = path.relative('visual/', p);
+
+    // eslint-disable-next-line @typescript-eslint/no-var-requires, global-require
+    return { path: p, data: require(`./${relativePath}`).scene };
+});
+
+const onlyScenes = scenes.filter((s) => s.data.only);
+const scenesToTest = onlyScenes.length ? onlyScenes : scenes;
+
+describe('Visual Tests', () =>
+{
+    scenesToTest.forEach((scene) =>
+    {
+        const defaultRenderers = {
+            canvas: false,
+            webgpu: true,
+            webgl: false,
```

Comment:
@GoodBoyDigital when you have both renderers working at the same time you can turn back on webgl here